classes/Validate.php

```diff
@@ -1286,4 +1287,16 @@ public static function isValidImapUrl($imapUrl)
 
         return true;
     }
+
+    /**
+     * Check the given string is a valid object class name
```

Comment:
```suggestion
     * Check the given string is a valid PHP class name
```
In order to remove any ambiguity ? 😉